controllers/article-controller.js

```diff
@@ -1,5 +1,16 @@
 const Article = require('../models/Article.js')
 
+exports.getAllArticles = async (req, res) => {
+  const {} = req.query
```

Comment:
If you're not using anything from the `req` then this line isn't necessary... yet